src/components/Form/Form.styled.ts

```diff
@@ -3,7 +3,6 @@ import styled from "styled-components";
 export const StyledForm = styled.form`
 	display: flex;
 	flex-direction: column;
-	justify-content: space-between;

```

Comment:
Did you delete this because of the blank space that appear between the layout and the top component? If you did then this doesn't fully solve the issue. The issue has to do more with this piece of code in Form.tsx
`useEffect(() => {
		setTopComponentHeight(topComponentRef.current?.offsetHeight);
	}, [topComponentRef.current?.offsetHeight]);`

Here the topComponent's offsetHeight is not being updated when the view changes, so it stays with the previous one, you can test this by opening the "Form with layout story" in a small screen size, then making it big until the nav links move to the left. It would look something like this: 
![image](https://user-images.githubusercontent.com/23409284/208525004-f5d09b33-b557-44d9-b03f-4a3157e698d6.png)

What I think might solve the issue is adding / changing the dependency of that useEffect to also include view, that way when it changes it also updates the height